index.html

```diff
@@ -118,7 +118,7 @@ <h2 class="project_title">Tonic</h2>
                   <li class="project_tag">javaScript</li>
                 </ul>
               </div>
-              <button type="button" class="view_project_btn">
+              <button type="button" class="view_project_btn proj_1">
                 See Project
               </button>
             </div>
```

Comment:
- [ ] Good job creating the dynamic code👍. Kindly note that already created HTML code for the projects list should be removed and all the HTML of the section is created when the page loads🙄

Reply:
hi @rotimiazeez ,Thanks for the review, but your message here is not clear on what changes you want me to make.

Reply:
This project only requires dynamicity of the Pop up modal window not the whole page @rotimiazeez